jlm/tooling/CommandLine.cpp

```diff
@@ -688,6 +688,8 @@ JlmOptCommandLineParser::ParseCommandLineArguments(int argc, const char * const
       cl::value_desc("file"));
 
   const auto statisticsDirectoryDefault = util::FilePath::TempDirectoryPath().Join("jlm");
+  statisticsDirectoryDefault.CreateDirectory();
```

Comment:
This should not be here, no?
This path is only the default output folder. If the user specifies a different one we should make that one instead. Also, I think it is cleaner to only make the directory if there is a pass that wants to write something there. See my comment above

Reply:
Removed.